lockbox-ios/Common/Extensions/UIViewController+.swift

```diff
@@ -22,35 +23,20 @@ protocol AlertControllerView {
                                 style: UIAlertControllerStyle)
 }
 
+protocol SpinnerAlertView {
+    func displaySpinner(_ dismiss: Driver<Void>, bag: DisposeBag)
+}
+
 extension UIViewController: StatusAlertView {
```

Comment:
😕 

Putting this on `UIViewController` is a little over-general. To be sure, not wrong, just… I'll keep reading.

Reply:
What if we want to throw up a spinner somewhere else? :p

Reply:
Right, I'm just seeing the other `StatusAlertView` and `AlertControllerView` on `UIViewController+.swift`.

If this was the first time you'd done this, I'd advise on an approach which `viewController.present(_ view: StatusAlertView, animated: Bool = false)`, with a convenience constructor on `StatusAlertView`.

However, this is an established pattern, so carry on.